internal/snykclient/snykclient.go

```diff
@@ -47,10 +48,19 @@ func NewSnykClient(c *http.Client, apiBaseURL, orgID string) *SnykClient {
 	}
 }
 
-func (t *SnykClient) CreateSBOMTest(ctx context.Context, sbom io.Reader) (*SBOMTest, error) {
+var JsonAPIPrefixBytes = []byte(`{"data":{"type":"sbom_test","attributes":{"sbom":`)
+var JsonAPISuffixBytes = []byte(`}}}`)
+
+func (t *SnykClient) CreateSBOMTest(ctx context.Context, sbomJSON io.Reader) (*SBOMTest, error) {
 	url := fmt.Sprintf("%s/rest/orgs/%s/sbom_tests?version=%s", t.apiBaseURL, t.orgID, sbomTestAPIVersion)
 
-	req, err := http.NewRequestWithContext(ctx, http.MethodPost, url, sbom)
+	jsonAPIReader := io.MultiReader(
+		bytes.NewReader(JsonAPIPrefixBytes),
+		sbomJSON,
+		bytes.NewReader(JsonAPISuffixBytes),
+	)
```

Comment:
can we simplify this by just concatenating a string, or using a `fmt.Sprintf` call with one string?

Reply:
I was avoiding having to load the entire file into memory just to add a prefix and suffix. This keeps the file as a reader allowing it to be efficiently streamed to the Snyk API regardless of file size.